README.md

```diff
@@ -23,6 +23,12 @@ To run
 Run the `acousticbrainz-gui` program. Select folders that contain music tagged
 with MusicBrainz identifiers. Click Analyze.
 
+*Note:* Since all available CPU threads will be used, consider lowering the priority of
```

Comment:
How do you do this on windows? (I don't have much experience with it). Do you use the task manager? or is there a way from setting it via a context menu?

Reply:
alternatively, is there a way that we can programatically set the priority of `streaming_extractor_music` processes a bit lower?

Reply:
You're spot-on: Open Task Manager > Details tab > right-click the GUI process > Set Priority > Low.

As mentioned in the OP, I did look in to programmatically setting the priority on the `QProcess`es used for running the stream extractor, but that appeared to require writing cross-platform code to do so, and I'm already out of my depth as-is!

I will note that setting the priority on `QThread`s looked simpler and cross-platform, but I don't know if it would be possible to invoke the extractor from the thread, or if the priority would ne inherited.  If we did do this, we might be doubling-up on threads (although, since one would be sitting idle waiting for the extractor to complete, maybe that is not so bad...).

Reply:
OK, thanks. I'm not very familiar with how QT works either, so this comment in the README is a good way to start. I'll have a dig around and see if I can work out a way to do it programatically too.